packages/workbench/src/main.single-spa.ts

```diff
@@ -7,12 +7,14 @@ import { singleSpaAngular, getSingleSpaExtraProviders } from 'single-spa-angular
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 import { singleSpaPropsSubject } from './single-spa/single-spa-props';
+import {defineCustomElements} from '../../shared-components/loader';
 
 if (environment.production) {
   enableProdMode();
 }
 
 // bootstrapApplication(AppComponent, appConfig).catch((err) => console.error(err));
+defineCustomElements(window);
```

Comment:
Is this needed? We defined the shared-components in root-config. In my opinion, we can use it everywhere on the page.

Reply:
You are probably right, but I need to test it.

Reply:
removed